### PR TITLE
Small but important grammar/editorial fixes

### PR DESCRIPTION
**Pull request by JAG-UK**
A bunch of small but important drafting fixes. 
Shouldn't be any controversial content changes in this one

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -113,39 +121,40 @@ As the Issuer of the Signed Statement and conformance to the Registration Policy
...
 Analogously, Transparent Statement Consumers rely on verifiable trustworthiness assertions associated with Transparent Statements and their processing provenance in a believable manner.
-If trust can be put into the operations that record Signed Statements in a secure, append-only log via online operations, the same trust can be put into the resulting transparent statement,
-issued by the Transparency Services and that can be validated in offline operations.
+If trust can be put into the operations that record Signed Statements in a secure, append-only log via _online_ operations, the same trust can be put into the resulting transparent statement issued by the Transparency Service that can be validated in _offline_ operations.
 
-The Transparency Services specified in this architecture can be implemented by various different types of services in various types of languages provided via various variants of API layouts.
+Breadth of access if critical so the Transparency Services specified in this architecture can be implemented by various different types of services in various types of languages provided via various variants of API layouts.
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:154]: > Breadth of access if critical so the Transparency 

Was this intended to be:
Breadth of access **is** critical so the

**Review by SteveLasker — COMMENTED** [review]
Thanks, @JAG-UK
Great edits, thank you. Overall LGTM, with one minor nit/clarification of wording.

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -113,39 +121,40 @@ As the Issuer of the Signed Statement and conformance to the Registration Policy
...
 
 A Transparency Services provides visibility into Signed Statements associated with various supply chains and their sub-systems.
-These Signed Statements (and corresponding Statement payload) are about the Artifacts produced by a supply chain.
+These Signed Statements (and corresponding Statement payload) make claims about the Artifacts produced by a supply chain.
 A Transparency Service endorses specific and well-defined metadata about these Artifacts that is captured in Statements.
 Some metadata is selected (and signed) by the Issuer, indicating, e.g., "who issued the Statement" or "what type of Artifact is described" or "what is the Artifact's version"; whereas additional metadata is selected (and countersigned) by the Transparency Services, indicating, e.g., "when was the Signed Statement about the Artifact registered in the Registry".
-Producing a Transparent Statement is also referred to as a form of notarization.
+Producing a Transparent Statement may be considered a form of notarization.
```
Comment (OR13) [inline draft-ietf-scitt-architecture.md:133]:
```suggestion
Producing a Transparent Statement might be considered a form of notarization.
```

Comment (OR13) [reply draft-ietf-scitt-architecture.md:133]: avoiding normative language, not sure if its as much of a deal as it is at W3C

**Review by OR13 — APPROVED** [review]
my nits can be ignored

**Comment by yogeshbdeshpande** [comment]
@JAG-UK I only see Orie Approval, so I would suggest we wait for other approvers, before merging this.

**Comment by henkbirkholz** [comment]
@JAG-UK, I'd like to see this PR presented in the group for feedback, if that is okay.

**Pull request merged**